Plausible/Sampleable.lean

```diff
@@ -109,23 +111,39 @@ class SampleableExt (α : Sort u) where
   proxy : Type v
   [proxyRepr : Repr proxy]
   [shrink : Shrinkable proxy]
+  proxyExpr? : Option (ToExpr proxy)
```

Comment:
Can you add a docstring explaining what this is for? I don't think you're using this in the way that I was trying to suggest on Zulip, but that doesn't mean it's a bad thing to have.